rskj-core/src/main/resources/reference.conf

```diff
@@ -233,4 +233,27 @@ scoring {
         # maximum punishment duration (in minutes, default = 1 week)
         maximum: 6000
     }
-}
\ No newline at end of file
+}
+
+prune {
+    # prune service could be enabled or not
+    # values: [true/false]
+    # default: false
```

Comment:
Change to `enabled = false` and remove the comment about the default. This file **is** the defaults.